sandbox/grommet-app/src/App.jsx

```diff
@@ -89,22 +90,14 @@ const App = () => {
                   setWorkspace={setWorkspace}
                   style={{ position: 'relative', zIndex: 1 }}
                 />
-                <Box
```

Comment:
Removed this Box with the height applied as it was causing cascading page height issues. Moved its purpose to be handled by the NavigationalSidebar